actionview/lib/action_view/template/error.rb

```diff
@@ -59,13 +59,16 @@ class Template
     class Error < ActionViewError #:nodoc:
       SOURCE_CODE_RADIUS = 3
 
-      attr_reader :original_exception
+      def initialize(template, message, backtrace)
+        super(message)
+        set_backtrace(backtrace)
```

Comment:
Possible to delegate these to `#cause`, or is that assigned after initialization?